mirage/config.js

```diff
@@ -85,6 +85,21 @@ export default function() {
 
   this.delete('/user_categories/:id');
 
+  this.get('/skills');
+  this.get('/user_skills');
+
+  // TODO: Make this work when relationships work
```

Comment:
It's not a many to many, though

It's a `one-to-many` `user` -> `userSkill` and a `one-to-many` `skill` -> `userSkill`. user -> skills would be a many to many relationship, but we don't have that on our model here. What we have is what mirage should already be supporting.